src/ws/index.js

```diff
@@ -96,8 +96,14 @@ function info ({ id }, callback) {
 }
 
 module.exports = {
-  _api,
   send,
   close,
   info,
 }
+
```

Comment:
Clever, the only problem I see with this solution (and it's a bit obscure) is that bc of this means of prototype manipulation, `_api` is no longer discoverable by looking at `arc.ws`. wdyt?